cli/commands/backend/delete/cli.go

```diff
@@ -0,0 +1,49 @@
+package delete
+
+import (
+	"github.com/gruntwork-io/terragrunt/cli/commands/common/runall"
+	"github.com/gruntwork-io/terragrunt/cli/commands/run"
+	"github.com/gruntwork-io/terragrunt/cli/flags"
+	"github.com/gruntwork-io/terragrunt/internal/cli"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+const (
+	CommandName = "delete"
+
+	BucketFlagName = "bucket"
+)
+
+func NewFlags(cmdOpts *Options, prefix flags.Prefix) cli.Flags {
+	tgPrefix := prefix.Prepend(flags.TgPrefix)
+
+	flags := cli.Flags{
+		flags.NewFlag(&cli.BoolFlag{
+			Name:        BucketFlagName,
+			EnvVars:     tgPrefix.EnvVars(BucketFlagName),
+			Usage:       "Delete the entire bucket.",
+			Hidden:      true,
```

Comment:
Please don't just hide this, but make it inaccessible either by removing it or commenting it out. It's not good that someone can accidentally delete their bucket without us adding safeguards like checking for bucket tags or something.

Reply:
I will leave it because it helps to return the remote storage to its initial state, it also helps to test the code properly. Otherwise, you will have to manually delete all the buckets, or change the bucket name in the config.
I added an error return.